apps/src/templates/instructions/codeReviewV2/Comment.jsx

```diff
@@ -179,99 +158,107 @@ class Comment extends Component {
     });
   };
 
-  flashErrorMessage = () => {
-    this.setState({displayError: true});
-    setTimeout(() => this.setState({displayError: false}), FLASH_ERROR_TIME_MS);
+  const flashErrorMessage = () => {
+    setDisplayError(true);
+    setTimeout(() => setDisplayError(false), FLASH_ERROR_TIME_MS);
   };
 
-  render() {
-    if (this.state.isDeleted) {
-      return null;
-    }
+  const {
+    commentText,
+    timestampString,
+    isFromTeacher,
+    isFromOlderVersionOfProject
+  } = comment;
 
-    const {
-      commentText,
-      timestampString,
-      isFromTeacher,
-      isFromOlderVersionOfProject
-    } = this.props.comment;
-
-    const {
-      hideResolved,
-      isUpdating,
-      isCommentResolved,
-      displayError
-    } = this.state;
+  if (isDeleted) {
+    return null;
+  }
 
-    return (
-      <div
-        style={{
-          ...styles.commentContainer,
-          ...((isFromOlderVersionOfProject || isCommentResolved) &&
-            styles.lessVisible)
-        }}
-      >
-        <div style={styles.commentHeaderContainer}>
-          {isCommentResolved && (
-            <i
-              className="fa fa-check-circle resolved-checkmark"
-              style={styles.check}
-            />
-          )}
-          {this.renderName()}
-          <span
-            style={styles.rightAlignedCommentHeaderSection}
-            className="comment-right-header"
-          >
-            <span style={styles.timestamp}>
-              {this.renderFormattedTimestamp(timestampString)}
-            </span>
-            {isUpdating ? (
-              <Spinner size="small" />
-            ) : (
-              <InlineDropdownMenu
-                selector={
-                  <img
-                    src={
-                      '/blockly/media/templates/instructions/codeReview/ellipsis.svg'
-                    }
-                    style={{height: '3px', display: 'flex'}}
-                  />
-                }
-              >
-                {this.getMenuItems()}
-              </InlineDropdownMenu>
-            )}
-          </span>
-        </div>
-        {!(isCommentResolved && hideResolved) && (
-          <div
-            className="code-review-comment-body"
-            style={{
-              ...styles.comment,
-              ...(isFromTeacher && styles.commentFromTeacher),
-              ...((isFromOlderVersionOfProject || isCommentResolved) &&
-                styles.lessVisibleBackgroundColor)
-            }}
-          >
-            {commentText}
-          </div>
+  return (
+    <div
+      style={{
+        ...styles.commentContainer,
+        ...((isFromOlderVersionOfProject || isCommentResolved) &&
+          styles.lessVisible)
+      }}
+    >
+      <div style={styles.commentHeaderContainer}>
+        {isCommentResolved && (
+          <i
```

Comment:
I know this was left over from v1, but let's use the FontAwesome component for icons